IOMC/ParticleGuns/src/CloseByParticleGunProducer.cc

```diff
@@ -25,9 +25,12 @@ CloseByParticleGunProducer::CloseByParticleGunProducer(const ParameterSet& pset)
   ParameterSet defpset;
   ParameterSet pgun_params = pset.getParameter<ParameterSet>("PGunParameters");
 
+  fControlledByEta = pgun_params.getParameter<bool>("ControlledByEta");
   fEnMax = pgun_params.getParameter<double>("EnMax");
   fEnMin = pgun_params.getParameter<double>("EnMin");
   fMaxEnSpread = pgun_params.getParameter<bool>("MaxEnSpread");
+  fEtaMax = pgun_params.getParameter<double>("MaxEta");
```

Comment:
Since you are (also) cleaning, the same style could be used for all Min/Max parameters: here `EtaMin` / `EtaMax` (and below also `PhiMin` / `PhiMax`)

Reply:
Hi! I tried to change from `EtaMin`/`EtaMax` to `MinEta`/`MaxEta`, I modified also the _cfi.py files, but when I try to test the workflow, I get 
```
----- Begin Fatal Exception 28-Dec-2021 13:07:11 CET-----------------------
An exception of category 'Configuration' occurred while
   [0] Constructing the EventProcessor
   [1] Constructing module: class=CloseByParticleGunProducer label='generator'
Exception Message:
MissingParameter: Parameter 'MinEta' not found.
----- End Fatal Exception -------------------------------------------------
```
Same thing with `Phi`.
Am I missing something?

Reply:
@waredjeb the message says that it looks from a "MinEta" parameter which is not able to find in the configuration. This makes me think that you did not modify in your patch the fillDescriptions method with the new parameter' names

Reply:
@perrotta The point is that there is not a fillDescription in the `CloseByParticleGunProducer`, that's why I modified the `cfi.py` manually

Reply:
Ah, ok. Then you did not modify the `pgun_params.getParameter<double>("MinEta") `call in the .cc code (or you did it but you did not recompile before testing)

Reply:
I found the problem, I had to change the parameter name in `IOMC/ParticleGuns/src/BaseFlatGunProducer.cc` as well.